include/nvector/nvector_parhyp.h

```diff
@@ -98,7 +98,7 @@ SUNDIALS_EXPORT void N_VSpace_ParHyp(N_Vector v, sunindextype *lrw,
                                      sunindextype *liw);
 SUNDIALS_EXPORT sunrealtype *N_VGetArrayPointer_ParHyp(N_Vector v);
 SUNDIALS_EXPORT void N_VSetArrayPointer_ParHyp(sunrealtype *v_data, N_Vector v);
-SUNDIALS_EXPORT void *N_VGetCommunicator_ParHyp(N_Vector v);
+SUNDIALS_EXPORT MPI_Comm N_VGetCommunicator_ParHyp(N_Vector v);
```

Comment:
Should this be `SUNComm`?
```suggestion
SUNDIALS_EXPORT SUNComm N_VGetCommunicator_ParHyp(N_Vector v);
```

Reply:
No, its an MPI only vector, there is no reason to use a SUNComm here (which would just be a typedef to MPI_Comm anyways). I guess we could, but I think I slightly prefer using MPI_Comm here.

Reply:
There does seem to be a mismatch in what I did though. Sometimes I used `SUNComm` when it really could be `MPI_Comm`. Does anyone have a preference or reason to choose one over the other?

Reply:
I don't have much of a preference either way, but I agree that consistency would be optimal.